lib/imageDisplay.ml

```diff
@@ -103,10 +103,19 @@ let print_image_file conf fname =
 let print_portrait conf base p =
   match Image.get_portrait conf base p with
   | Some (`Path path) ->
-      Result.fold ~ok:ignore
-        ~error:(fun _ -> Hutil.incorrect_request conf)
-        (print_image_file conf path)
-  | Some (`Url _) | None -> Hutil.incorrect_request conf
+      if Mutil.start_with "http" 0 path then Hutil.incorrect_request conf
```

Comment:
Do we need this check?
a Path should not be an url... does it happen because of a bad entry in the database or is it an issue with get_portrait?
maybe this check should be in `get_portrait`?